Agoric/Agoric
# feat: read only smart wallet

refs: https://github.com/Agoric/agoric-sdk/issues/3995

Testing steps:
- `cd agoric-sdk`
- `yarn && yarn build`
- `cd agoric-sdk/packages/cosmic-swingset`
- `make`
- `make scenario2-setup`
- `make scenario2-run-chain`
- `make scenario2-run-client`
- Get address from `cat t1/8000/ag-cosmos-helper-address`
- `agoric open`

Select "Read-only Address":
![image](https://user-images.githubusercontent.com/8848650/178076338-806b9520-cf4c-4653-8022-a00e78a0b216.png)

Then all the purses, issuers, etc. should load in a read-only state.
